src/components/material/ReviewInfomedia.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {
+  InfomediaReview,
+  useGetInfomediaQuery
+} from "../../core/dbc-gateway/generated/graphql";
+import { useText } from "../../core/utils/text";
+import ReviewHearts from "./ReviewHearts";
+import ReviewMetadata from "./ReviewMetadata";
+
+export interface ReviewInfomediaProps {
+  review: InfomediaReview;
+}
+
+const ReviewInfomedia: React.FC<ReviewInfomediaProps> = ({ review }) => {
+  const { id } = review;
+  const { data } = useGetInfomediaQuery({
+    id
+  });
+  const t = useText();
+  return (
+    <li className="review text-small-caption">
+      {(review.author || review.date) && (
+        <ReviewMetadata author={review.author} date={review.date} />
+      )}
+      {review.rating && <ReviewHearts amountOfHearts={review.rating} />}
+      {data?.infomedia.article?.headLine && (
+        <div className="review__headline mb-8">
+          {data.infomedia.article.headLine}
+        </div>
+      )}
+      {data?.infomedia.error && (
+        <div className="review__headline mb-8">
+          {data?.infomedia.error === "BORROWER_NOT_LOGGED_IN"
+            ? t("loginToSeeReviewText")
+            : t("cantViewReviewText")}
+        </div>
+      )}
+      {data?.infomedia.article?.text && (
+        <p className="review__body mb-8">{data?.infomedia.article?.text}</p>
+      )}
+      {review.origin && (
+        <a href={review.origin} className="link-tag text-small-caption mb-8">
+          {`${review.author}${
+            review.author && review.date ? ", " : ""
+          }${review.date?.split("-").reverse().join(".")}`}
```

Comment:
Consider whether proper date handling can replace string manipulation.

I would expect something like creating a `Date` object from the string and then formatting the resulting object according to our wishes. We may need a library to help us out here.

I recommend https://date-fns.org/ but @spaceo may also have an opinion here.

Whether you do please move handling into a separate function as this occurs multiple places.

Reply:
Yes. I think you should use dayjs as described earlier. But I also think that you could use `ReviewMetadata` in both places. It seems like the same thing is happening with checking author and date and formatting date.

Reply:
Perf, yes ;) Thanks for the suggestion 💯 Fixing it.

Reply:
Consider if you can use the the new helper here.

Reply:
Snap, forgot to use it here too 🎖️ 🎖️ 🎖️ Thx